### PR TITLE
CVX Vaccine Groups naming convention for VC types

### PR DESCRIPTION
**Pull request by jmandel**

### CODE REVIEW

**Review by p2-apple — APPROVED** [review]

**Review by p2-apple — CHANGES_REQUESTED** [review]
Pending discussion in https://github.com/HL7/fhir-shc-vaccination-ig/pull/173#issuecomment-898603663

**Review by p2-apple — APPROVED** [review]

**Comment by woodynj** [comment]
Good

**Review by woodynj — APPROVED** [review]

**Review by jpp9 — COMMENTED** [review]
i like it. we'd support this behavior in CommonHealth app.

**Review by p2-apple — APPROVED** [review]
This is great, yes!

**Comment by vitorpamplona** [comment]
We also agree with these changes.

File: docs/changelog.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.3.0
```
Comment (christianpaquin) [inline docs/changelog.md:3]: This needs to be updated to 1.4.0, since 1.3.0 now already exists

Comment (jmandel) [reply docs/changelog.md:3]: Good call-out! Will adjust before merging

File: docs/index.md [hunk 2]
```diff
@@ -326,22 +326,54 @@ A SMART on FHIR Server capable of issuing VCs according to this specification SH
...
-The following parameters are optional; clients MAY include them in a request, and servers MAY ignore them if present.
+```json
+{
+  "resourceType": "Parameters",
+  "parameter": [{
+    "name": "credentialType",
+    "valueUri": "Immunization"
+  }, {
+    "name": "credentialValueSet",
+    "valueUri": "https://terminology.smarthealth.cards/ValueSet/monkeypox-cvx"
```
Comment (masnick) [inline docs/index.md:373]:
```suggestion
    "valueUri": "https://terminology.smarthealth.cards/ValueSet/immunization-orthopoxvirus-all"
```

**Pull request merged**